### DISCUSSION TITLE
Unable to make a mesh watertight

### DISCUSSION THREAD

**Discussion by sree9578** [CGAL/cgal#7468] [General]
I have obj file with polygon meshes 
I am trying unwrap in blender , as the faces are not connected properly they are not distributed properly for UV coords.
I am also not able to do certain operation as the mesh is not watertight 

I tried stitchborders form meshlab examples , still the mesh is not unwarped properly , guide me on steps to make mesh watertight , and unwrap properly

**Comment by sloriot** [comment]
Could you share the mesh?
